components/menu/src/auro-menu.js

```diff
@@ -185,7 +185,7 @@ export class AuroMenu extends LitElement {
       this.items.forEach((item) => {
```

Comment:
looks like `this.selected = false;` would be needed.

Reply:
`this` would be the `auro-menu` component, but not the `menuoption`.

Should it be `item.selected = false;`?

```
resetOptionsStates() {
  this.optionSelected = undefined;
  if (this.items) {
    this.items.forEach((item) => {
      item.classList.remove('active');
      item.removeAttribute('selected');
      item.selected = false;
    });
  }
}
````

Reply:
Oh my comment was mixed up. I think `item.removeAttribute('selected');` would do it enough.